lib/LocalizationProvider.js

```diff
@@ -90,7 +109,27 @@ LocalizationProvider.prototype._pluralizationRulesCache = null;
  * @param {Object} context Component context.
  */
 LocalizationProvider.prototype.getCurrentLocale = function (context) {
-	return context.cookie.get(this._cookieName) || this._defaultLocale;
+	return context.cookie.get(this._cookieConfig.name) || this._defaultLocale;
+};
+
+/**
+ * Changes current locale value.
+ * @param {string} locale Locale name (i.e. en, en-us, ru etc).
+ * @param {Object} context Component context.
+ */
+LocalizationProvider.prototype.changeLocale = function (locale, context) {
+	var expireDate = new Date((new Date()).getTime() +
+			this._cookieConfig.maxAge * 1000);
+
+	this._cookieConfig.key = this._cookieConfig.name;
+	this._cookieConfig.value = locale;
+	this._cookieConfig.expires = expireDate.toUTCString();
```

Comment:
One more thing I've found here: you should not use `.toUTCString()`, this parameter should be a `Date` instance.
You can found parameter types in jsDoc [here](https://github.com/catberry/catberry/blob/4.6.4/docs/index.md#cookie).